models/intermediate/shopify__inventory_level__aggregates.sql

```diff
@@ -0,0 +1,103 @@
+with order_lines as (
+
+    select *
+    from {{ var('shopify_order_line') }}
+),
+
+fulfillment as (
+
+    select *
+    from {{ var('shopify_fulfillment') }}
+),
+
+orders as (
+
+    select *
+    from {{ var('shopify_order') }}
+    where not coalesce(is_deleted, false)
+), 
+
+{% if fivetran_utils.enabled_vars(vars=["shopify__using_order_line_refund", "shopify__using_refund"]) %}
```

Comment:
With the updates you applied to the [refund temp](https://github.com/fivetran/dbt_shopify_source/blob/feature/package-revamp/models/tmp/stg_shopify__refund_tmp.sql) model, is this conditional needed still?

Reply:
It looks like the [order line refund temp](https://github.com/fivetran/dbt_shopify_source/blob/feature/package-revamp/models/tmp/stg_shopify__order_line_refund_tmp.sql) model hasn't gotten the same update. If both receive the update to swap out the variable declaration with the `does exist` logic, I think we can remove the conditionals from these end models altogether.

What are your thoughts?

Reply:
are we trying to completely get rid of these variables? because to remove the conditionals, we'd need to remove  `{{ config(enabled=var('shopify__using_refund', True)) }}` from all of the non-tmp refund models, effectively rendering them useless

this sounds good to me, but just wanna confirm!

Reply:
Yeah let's go forward with removing those variables across the board. The refunds and adjustments (to my understanding) are less a feature that is turned off and on and is instead just populated in the source once one comes through. Therefore, the variable doesn't make much sense and would be more appropriate in my opinion to be a default behavior of the package to determine for users.

Let's be sure to document this in the DECISIONLOG so customers are aware of this default behavior and aren't confused by the empty models if they don't have adjustments or refunds.

Reply:
updated here and the source package!